pkg/apiserver/debugapi/http_client.go

```diff
@@ -81,43 +81,63 @@ type tidbImplement struct {
 	Client *tidb.Client
 }
 
-func (impl *tidbImplement) Get(payload *endpoint.ResolvedRequestPayload) (*httpc.Response, error) {
-	return impl.Client.
+func (impl *tidbImplement) Get(payload *endpoint.ResolvedRequestPayload) (*http.Response, error) {
```

Comment:
Why do we have `http.Response` sometimes and `httpc.Response` sometimes? Which one should I use in different cases?